### DISCUSSION TITLE
useCalendar weekday abbreviations

### DISCUSSION THREAD

**Discussion by jake-chapman-mark43** [adobe/react-spectrum#3940] [Q&A]
All code examples I've found only display the weekdays as single letters, S M T W T F S. 

Is there a way to get returned the weekdays from `useCalendarGrid` as a few different options instead of single letter abbreviations?

Would like to get `Sun Mon Tue Wed Thu Fri Sat` instead.

**Comment by snowystinger** [comment]
See https://github.com/adobe/react-spectrum/issues/3925
